src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,23 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = null;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Invalid login or password for login: {}", user.getLogin());
```

Comment:
The log message should include the login that was attempted rather than the one from the user object which might be null if authentication failed. Consider logging the actual login string used in the authentication attempt.